package/upgrade/upgrade_node.sh

```diff
@@ -294,6 +301,53 @@ clean_rke2_archives() {
     done
 }
 
+convert_nodenetwork_to_vlanconfig() {
+  [[ $UPGRADE_PREVIOUS_VERSION != "1.0.3" ]] && return
+
+  # Don't need to convert nodenetwork if harvester-mgmt is used as vlan interface in any nodenetwork
+  [[ $(kubectl get nodenetwork -o yaml | yq '.items[].spec.nic | select(. == "harvester-mgmt")') ]] && return
+
+  local name=${HARVESTER_UPGRADE_NODE_NAME}-vlan
+
+  nn=$(kubectl get nn "$name" -o yaml)
+  vlan_interface=$(echo "$nn" | yq '.spec.nic')
```

Comment:
The usage of `yq '.spec.nic'` loosk to be wrong, and there are many similar usage, please double check

```
$cat ./99_custom_v103_11.yaml | yq '.spec.nic'

Error: unknown command ".spec.nic" for "yq"
```

even cat a file as input for `yq` is not working
```
$cat ./99_custom_v103_11.yaml | yq e '.spec.nic'
Error: open .spec.nic: no such file or directory
```
not working
```
$ yq '.stages.initramfs[0]' ./99_custom_v103_11.yaml 
Error: unknown command ".stages.initramfs[0]" for "yq"

Run 'yq --help' for usage.
```

only working as :
```
$ yq e '.stages.initramfs[0]' ./99_custom_v103_11.yaml
```

Reply:
It's due to the `yq` version. Refer to yq doc https://mikefarah.gitbook.io/yq/#notice-for-v4.x-versions-prior-to-4.18.1.

Reply:
The yq version in harvester 1.1.0 is bigger than 4.18.1.
```
bm-harv2:/home/rancher # yq -V
yq (https://github.com/mikefarah/yq/) version 4.24.5
```